apps/editor.planx.uk/src/pages/FlowEditor/components/Settings/Flow/Disclaimers/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import GuidanceDisclaimer from "./components/Guidance";
+import ResultDisclaimer from "./components/Result";
+
+const Disclaimers: React.FC = () => (
+  <>
+    <ResultDisclaimer />
+    <GuidanceDisclaimer />
```

Comment:
In the future, this a good candidate for "conditional settings" - eg hide for submission services. But leaving always-visible for now!